docs/general/plugin-ca.md

```diff
@@ -0,0 +1,194 @@
+[Return to Project Root](../../README.md)
+
+# Plug in CA Certificates
+Istio [documentation](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/) is covering how to plug in user generated certificates to be used by the Istio CA but it's not describing a use case where it's necessary to switch from Istio CA generated self-signed certificates to user generated certificates without any traffic disruptions even with strict mTLS enabled. This missing use case is covered here.
+
+## Switching from Istio CA generated self-signed certificates
+By default the Istio CA generates a self-signed root certificate and key and uses them to sign the workload certificates. This is not recommended in production but there might be cases where it's used already and we need to switch to better certificate management method. It's a simple [task](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/) to be done during a maintenance window where the traffic disruptions are allowed. If the same task must be done outside of the maintenance window without any traffic disruptions, the procedure is more complex.
```

Comment:
> This is not recommended in production

Why is it not recommended for production?

Reply:
For security reason, root CA private key should be stored on externally. With this default configuration, it's possible to extract it from istio-ca-secret secret. Should I extend this with explanation?